justfile

```diff
@@ -49,3 +48,6 @@ print-staging-report: (_run_ops_bin 'print_staging_report')
 check-genesis-integrity: (_run_ops_bin 'check_genesis_integrity')
 
 codegen: (_run_ops_bin 'codegen')
+
+inflate-config SHORTNAME FILENAME:
```

Comment:
`inflate-config` is a confusing term to me. Could we use `create-config` instead?